### ISSUE TITLE
Repeating last seek with ; and ,

### ISSUE THREAD

**Issue by saltydressage** [goldfeld/vim-seek#7]
Good extension.  A sort of 'why didn't I think of that!' idea...I like it.

I'd REALLY like to see this:  Repeating the last seek with ;

**Comment by goldfeld** [comment]
Glad you like it! Yes repeating is also a major priority for my own use, I'll try to get something working within the week. Thanks for the feedback.

**Comment by Aaron2Ti** [comment]
+1 for this

thanks for the great idea

**Comment by goldfeld** [comment]
I've looked high and low for a solution for this that would just work, but came up empty. Would simply allowing people to rebind their keys to SeekRepeatLast be an acceptable solution? For instance you could `nnoremap ; <silent> :SeekRepeatLast<CR>` and `nnoremap <silent> , :SeekBackRepeatLast<CR>`. 

To also make it still work with f/F/t/T I would also provide a SeekFindWrap command, so you would remap keys f/F/t/T like so `nnoremap <silent> f :SeekFindWrap f<CR>`, and so the result is that your find keys would work as normal, and your ; and , (or whatever you use for repeating) would also work as normal (repeating finds), with seeks also entering the fray, so ; and , would repeat whichever was the last command among seeks and finds.

**Comment by freeo** [comment]
If you supply the vimrc lines for the default ; and , keys then it will be fine I guess!

**Comment by saltydressage** [comment]
Yeah I concur with eyetracker.  That should work if the usage is transparent to the end user and you provide examples of what to do, or better yet just do it in the plugin and document it :)

**Comment by justinmk** [comment]
@jeffes @eyetracker @Aaron2Ti  [vim-sneak](https://github.com/justinmk/vim-sneak) supports repeat-motion via `;` and `,` out-of-the-box, and you can change the mappings easily.
